README.md

```diff
@@ -49,7 +49,7 @@ The site should now be accessible on `localhost:4000` on the host, and can be mo
 
 ### Building for Deployment
```

Comment:
what should a developer do, if in the future he/she is going to create or re-use a .js/.css file?